### ISSUE TITLE
Vendor contact form improvement suggestion

### ISSUE THREAD

**Issue by davidsherlock** [easydigitaldownloads/vendd#221]
I agree a modal box is a bad idea, however I have a better suggestion for the vendor contact form to position it better within the vendor page so it's not just dumped at the bottom of the page.

My idea was to use tabs, which would allow for the contact form to be hidden initially until clicked. Here is a screenshot of my rough implementation (still reworking the look of it).

The tabbed approach would have a few advantages such as allowing additional tabs for a larger about page, vendor public wishlists/collections, etc. Just an idea that might be worth considering!

![screenshot 2017-05-26 12 22 42](https://cloud.githubusercontent.com/assets/22668877/26492915/998f728a-420e-11e7-99b8-047c48fc851a.png)

**Comment by SDavisMedia** [comment]
@davidsherlock full custom or are you using a library to implement a tabbed interface? Requires a page load or switches tabs in place?

**Comment by davidsherlock** [comment]
Was just a small bit of JavaScript, HTML and css mainly, no custom libraries or alike. It's possible a better way of doing it exist but I wanted a quick solution! Just feel it's a better layout for the profile page so the contact form isn't easily missed.

**Comment by SDavisMedia** [comment]
I agree totally. I experimented with a few other layouts but ultimately chose to put it at the bottom. I might revisit again eventually. It's not high priority at the moment.

**Comment by spencerfinnell** [comment]
If tabs are implemented you should make sure they can be linked to directly and the relevant content is shown.

**Comment by SDavisMedia** [comment]
@spencerfinnell no doubt about it. Learned that lesson on EDD site's account page.
